tox.ini

```diff
@@ -263,7 +264,8 @@ max-line-length = 88
 python =
   2.6: py26,custom
   2.7: py27,flake8,pylint,custom
-  3.5: coveralls,molecule,custom
+  3.5: coveralls,custom
   3.6: py36,black,custom
   3.7: py37,custom
   3.8: py38,custom
+  3.8-dev: py38,custom
```

Comment:
This does not seem to be needed, py38 already runs when the 3.8-dev version is installed:
https://travis-ci.com/linux-system-roles/network/jobs/281802888#L215

Reply:
Ok, I will remove it. Running custom commands only for stable Python versions should be enough.

Reply:
Done in 4b05c20.

Reply:
Even without the line, the custom command will still run with the python 3.8 development version because 3.8 in this config also matches the development version AFAICS.

Reply:
> Even without the line, the custom command will still run with the python 3.8 development version.

I do not see the custom command in the [log](https://travis-ci.com/linux-system-roles/network/jobs/282947052). However, it should be imho enough that the custom command runs in stable Python versions only.